release/release_tests.yaml

```diff
@@ -4211,7 +4211,7 @@
 
   run:
     timeout: 18000
-    script: python learning_tests/run.py --yaml-sub-dir=ppo --framework=tf
+    script: python learning_tests/run.py --yaml-sub-dir=ppo/old_stack --framework=tf
```

Comment:
This only fixes an inaccuracy that led to us attempting to run the new stack with `framework=tf`, which ultimately makes it so that we don't run with rl modules at all.

Reply:
:+1: